app/editor/src/features/admin/topics/Columns.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import {
+  Button,
+  ButtonVariant,
+  FieldSize,
+  getEnumStringOptions,
+  ITableHookColumn,
+  ITopicModel,
+  OptionItem,
+  Select,
+  TopicTypeName,
+} from 'tno-core';
+
+export const Columns = (
+  onClick: (event: any) => {},
+  handleSubmit: (values: ITopicModel) => Promise<void>,
+): ITableHookColumn<ITopicModel>[] => {
+  const [topicModel, setTopicModel] = React.useState<ITopicModel>();
```

Comment:
You can only use state within a Component or a Hook.  You've tricked the parser by capitalizing this function.

Reply:
The trick seems to be working fine in this case :) 
In the meantime, I'll see if we could find a better way to address this...

Reply:
You don't need to trick the parser, you should just follow the standard.  If you want to treat this like a function then create a Hook by renaming as `useColumns` and declare it like we do all other hooks.

Reply:
Updated, thanks